### PR TITLE
Telemetry-enable: Corrects attributions, formatting, hyperlink.

### PR DESCRIPTION
**Pull request by mvincerx**
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/maintenance/telemetry-enable.rst [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
-Enable and disable telemetry in Clear Linux
-###########################################
+Enable and disable telemetry in Clear Linux* OS
+###############################################
 
-|CL| includes a telemetry solution as part of the OS that records events
+|CL-ATTR| includes a telemetry solution as part of the OS that records events
 of interest and reports them back to the development team via the telemetrics
 daemon, **telemd**. This functionality is maintained in the
```
[the remark this answers is not in the document]
Comment (mvincerx) [reply source/clear-linux/guides/maintenance/telemetry-enable.rst]: Revised line adding new client daemons in latest push.

**Comment by mvincerx** [comment]
@bd-dean. Beat me to it! Thanks.

**Comment by mvincerx** [comment]
Understood. @krisdale has been overwhelmed with requests today. Patience, please. Needs TCS review too.

File: source/clear-linux/guides/maintenance/telemetry-enable.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .. _telemetry-enable:
 
-Enable and disable telemetry in Clear Linux
-###########################################
+Enable and disable telemetry in Clear Linux* OS
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/telemetry-enable.rst]: Replace "Clear Linux* OS" with CL-ATTR sub.

File: source/clear-linux/guides/maintenance/telemetry-enable.rst [hunk 3]
```diff
@@ -42,12 +42,12 @@ To start telemetry on your system, run the following command:
```
Comment (krisdale) [inline source/clear-linux/guides/maintenance/telemetry-enable.rst]: At lines 38 and 53, section title is not in sentence case ("Telemetry" in section title should be lower case).

**Pull request merged**